CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ set(IGN_MATH_VER ${ignition-math6_VERSION_MAJOR})
 
 #--------------------------------------
 # Find ignition-common
-ign_find_package(ignition-common4 REQUIRED VERSION 4.1)
+ign_find_package(ignition-common4 REQUIRED COMPONENTS profiler VERSION 4.1)
```

Comment:
a nightly build failure is a good reminder to update the Debian metadata with this new dependency

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ign-gui6-debbuilder&build=190)](https://build.osrfoundation.org/job/ign-gui6-debbuilder/190/) https://build.osrfoundation.org/job/ign-gui6-debbuilder/190/

~~~
CMake Error at /usr/share/cmake/ignition-cmake2/cmake2/IgnConfigureBuild.cmake:59 (message):
  -- 	Missing dependency [ignition-common4] (Components: profiler)
~~~

Reply:
We should add these to the buildfarmer's dashboard :smirk:

Reply:
I will be learning about that soon

Reply:
we can start by adding them to our Jenkins views: https://github.com/ignition-tooling/release-tools/pull/491